src/components/ArticleNavigation/ArticleNavigation.scss

```diff
@@ -0,0 +1,60 @@
+@use '~@gravity-ui/page-constructor/styles/variables.scss' as pcVariables;
+@use '../../variables.scss';
+
+$block: '.#{variables.$ns}article-navigation';
+
+#{$block} {
+    display: flex;
+    gap: var(--g-spacing-6);
+    @media (max-width: map-get(pcVariables.$gridBreakpoints, 'md') - 1) {
+        gap: var(--g-spacing-5);
+    }
+    &__content {
+        padding: var(--g-spacing-2) 0;
+        overflow: hidden;
+        width: 100%;
+        @media (max-width: map-get(pcVariables.$gridBreakpoints, 'md') - 1) {
+            padding: var(--g-spacing-3) 0;
+        }
+        &-title {
+            @media (max-width: map-get(pcVariables.$gridBreakpoints, 'md') - 1) {
+                display: none;
+            }
+        }
+    }
+    &__button {
+        cursor: pointer;
+        display: flex;
+        gap: var(--g-spacing-3);
+        width: 50%;
+        border-radius: 16px;
+        border: 1px solid var(--g-color-sfx-fade);
+        padding: var(--g-spacing-1);
+        justify-content: space-between;
+        &_reverse {
+            flex-direction: row-reverse;
+        }
+        &_right {
+            margin-left: auto;
+            padding-left: var(--g-spacing-5);
+        }
+        &_left {
+            margin-right: auto;
+            padding-right: var(--g-spacing-5);
+        }
+        &-icon {
+            flex: none;
+            width: 56px;
+            height: 100%;
+            border-radius: 12px;
+            background: var(--g-color-base-generic);
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            color: var(--g-color-text-primary);
+            @media (max-width: map-get(pcVariables.$gridBreakpoints, 'md') - 1) {
+                width: 40px;
```

Comment:
It can be simplified using padding. Since your icon is square, just add the necessary amount of padding to it. Specifying the width in pixels is usually unnecessary